### ISSUE TITLE
When predicting, is the batch size and the running CPU process the same?

### ISSUE THREAD

**Issue by HyungSik-Jo** [deepchem/deepchem#3251]
I've been testing GCNmodel in several directions, and I've got a question.

After training and saving the GCN model, the model is called and used.

When loading the model and making predictions, it seems that the CPU is allocated as much as the batch size specified in htop.

As far as I know, changing the batch size changes the memory allocation of the GPU.

Is there a relationship between batch size and CPU running process?

save code :
`
device = torch.device('cuda:0')
val_metric = dc.metrics.Metric(dc.metrics.mean_squared_error, mode="regression")
vc_valid = ValidationCallback(val_dataset, interval=100, metrics=[val_metric])

model = GCNModel(n_tasks=1, mode='regression', number_atom_features=32, learning_rate=0.0001, dropout=0.2, batch_size=32, device=device, model_dir=model_dir)

model.fit(train_dataset, nb_epoch=20, callbacks=[vc_valid])
`

load code 
`
model = GCNModel(n_tasks=1, mode='regression', number_atom_features=32, learning_rate=0.0001, batch_size=32, device=device, model_dir=model_path)
model.restore()
`

htop images
![image](https://user-images.githubusercontent.com/78404350/220536918-debb0456-ec05-436b-9343-29c200ba4d8d.png)
